src/util/BlankNodeManager.h

```diff
@@ -64,6 +70,13 @@ class BlankNodeManager {
     explicit LocalBlankNodeManager(BlankNodeManager* blankNodeManager);
     ~LocalBlankNodeManager();
 
+    // No copy, as the managed blocks shall not be duplicated.
+    LocalBlankNodeManager(const LocalBlankNodeManager&) = delete;
+    LocalBlankNodeManager& operator=(const LocalBlankNodeManager&) = delete;
+
+    LocalBlankNodeManager(LocalBlankNodeManager&&) = default;
+    LocalBlankNodeManager& operator=(LocalBlankNodeManager&&) = default;
```

Comment:
That is a good point of codecov:
Add a unit test, that moving of the local blank node managers does the freeing of the blocks still only once and at the right place. That makes us robust on ou assuptions on the moving of `std::vector`